packages/api/src/config/brevo-module.config.ts

```diff
@@ -6,10 +6,16 @@ import { BrevoContactFilterAttributesInterface } from "../types";
 
 export interface BrevoModuleConfig {
     brevo: {
-        apiKey: string;
+        getBrevoConfig: (scope: EmailCampaignScopeInterface) => {
```

Comment:
IMO getConfig is enough since its already in `brevo`. Could also be `resolveConfig`

Reply:
you are right, I went with `resolveConfig`. Sounds better 
[af37fd8](https://github.com/vivid-planet/comet-brevo-module/pull/53/commits/af37fd8f6b6250b854378548766c47be5ef8b998)